packages/appium/test/helpers.js

```diff
@@ -0,0 +1,35 @@
+import path from 'path';
+import {insertAppiumPrefixes} from '../lib/utils';
+import findUp from 'find-up';
+import getPort from 'get-port';
+
+const TEST_HOST = 'localhost';
```

Comment:
I would rather prefer 127.0.0.1
Some hosts may resolve to ipv6 address by default, which we don't want

Reply:
actually, I think I'm going to leave it, since `localhost` is hardcoded throughout the tests and it will be a pretty sizable diff.

Reply:
see https://github.com/appium/appium/issues/15399